neo/SmartContract/ContractAbi.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace Neo.SmartContract
+{
+    /// <summary>
+    /// For technical details of ABI, please refer to NEP-3: NeoContract ABI. (https://github.com/neo-project/proposals/blob/master/nep-3.mediawiki)
+    /// </summary>
+    public class ContractAbi
+    {
+        /// <summary>
+        /// Hash is the script hash of the contract. It is encoded as a hexadecimal string in big-endian.
+        /// </summary>
+        public UInt160 Hash { get; set; }
+
+        /// <summary>
+        /// Entrypoint is a Method object which describe the details of the entrypoint of the contract.
+        /// </summary>
+        public string EntryPoint { get; set; }
```

Comment:
Should be `ContractMethodWithReturnDefinition` I think.

Reply:
The entry point will not be defined in methods as well?

Reply:
No. the entry point is defined in `entrypoint`.

`methods` now only contains the methods that can be invoked with `operation`.